sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -52,6 +52,8 @@ import {
   OFFERING_STUDY_PERIOD_MAX_DAYS,
   OFFERING_STUDY_PERIOD_MIN_DAYS_FULL_TIME,
   OFFERING_STUDY_PERIOD_MIN_DAYS_PART_TIME,
```

Comment:
The constants `OFFERING_STUDY_PERIOD_MIN_DAYS_FULL_TIME` and `OFFERING_STUDY_PERIOD_MIN_DAYS_PART_TIME` are no longer required and need to be removed from all places.